File: Vicious.php
<!--────────────────────────────────────────────────────────────────────────────────────────────────────────-->
<!--─██████──██████──██████████──██████████████──██████████──██████████████──██████──██████──██████████████─-->
<!--─██░░██──██░░██──██░░░░░░██──██░░░░░░░░░░██──██░░░░░░██──██░░░░░░░░░░██──██░░██──██░░██──██░░░░░░░░░░██─-->
<!--─██░░██──██░░██──████░░████──██░░██████████──████░░████──██░░██████░░██──██░░██──██░░██──██░░██████████─-->
<!--─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██░░██─────────-->
<!--─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██░░██████████─-->
<!--─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██░░░░░░░░░░██─-->
<!--─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██████████░░██─-->
<!--─██░░░░██░░░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──────────██░░██─-->
<!--─████░░░░░░████──████░░████──██░░██████████──████░░████──██░░██████░░██──██░░██████░░██──██████████░░██─-->
<!--───████░░████────██░░░░░░██──██░░░░░░░░░░██──██░░░░░░██──██░░░░░░░░░░██──██░░░░░░░░░░██──██░░░░░░░░░░██─-->
<!--─────██████──────██████████──██████████████──██████████──██████████████──██████████████──██████████████─-->
<!--────────────────────────────────────────────────────────────────────────────────────────────────────────-->

<?php
class Person
{
  private $name;
  private $age;

  // Constructor
  public function __construct($name, $age)
  {
    $this->name = $name;
    $this->age = $age;
  }

  // Getters and setters
  public function getName()
  {
    return $this->name;
  }

  public function getAge()
  {
    return $this->age;
  }

  public function setName($name)
  {
    $this->name = $name;
  }

  public function setAge($age)
  {
    $this->age = $age;
  }

  // Method to greet
  public function greet()
  {
    echo "Hello, my name is " . $this->name . " and I am " . $this->age . " years old.<br>";
  }
}

// Create an instance of the Person class
$person = new Person("John Doe", 30);

// Call methods and access properties
$person->greet();
echo "Name: " . $person->getName() . "<br>";
echo "Age: " . $person->getAge() . "<br>";

// Inheritance
class Student extends Person
{
  private $studentId;

  public function __construct($name, $age, $studentId)
  {
    parent::__construct($name, $age);
    $this->studentId = $studentId;
  }

  public function getStudentId()
  {
    return $this->studentId;
  }

  public function setStudentId($studentId)
  {
    $this->studentId = $studentId;
  }

  public function greet()
  {
    parent::greet();
    echo "I am a student with ID: " . $this->studentId . "<br>";
  }
}

// Create an instance of the Student class
$student = new Student("Jane Smith", 25, "S12345");

// Call methods and access properties
$student->greet();
echo "Name: " . $student->getName() . "<br>";
echo "Age: " . $student->getAge() . "<br>";
echo "Student ID: " . $student->getStudentId() . "<br>";
?>

File: Vicious.rs
/**─────────────────────────────────────────────────────────────────────────────────────────────────────────**/
/** ─██████──██████──██████████──██████████████──██████████──██████████████──██████──██████──██████████████─**/
/** ─██░░██──██░░██──██░░░░░░██──██░░░░░░░░░░██──██░░░░░░██──██░░░░░░░░░░██──██░░██──██░░██──██░░░░░░░░░░██─**/
/** ─██░░██──██░░██──████░░████──██░░██████████──████░░████──██░░██████░░██──██░░██──██░░██──██░░██████████─**/
/** ─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██░░██─────────**/
/** ─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██░░██████████─**/
/** ─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██░░░░░░░░░░██─**/
/** ─██░░██──██░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──██████████░░██─**/
/** ─██░░░░██░░░░██────██░░██────██░░██────────────██░░██────██░░██──██░░██──██░░██──██░░██──────────██░░██─**/
/** ─████░░░░░░████──████░░████──██░░██████████──████░░████──██░░██████░░██──██░░██████░░██──██████████░░██─**/
/** ───████░░████────██░░░░░░██──██░░░░░░░░░░██──██░░░░░░██──██░░░░░░░░░░██──██░░░░░░░░░░██──██░░░░░░░░░░██─**/
/** ─────██████──────██████████──██████████████──██████████──██████████████──██████████████──██████████████─**/
/** ────────────────────────────────────────────────────────────────────────────────────────────────────────**/

 // Define a struct
struct Person {
    name: String,
    age: u32,
}

impl Person {
    // Define a method on the Person struct
    fn greet(&self) {
        println!("Hello, my name is {} and I am {} years old.", self.name, self.age);
    }
}

// Implement the Default trait for the Person struct
impl Default for Person {
    fn default() -> Self {
        Person {
            name: String::from("John Doe"),
            age: 30,
        }
    }
}

// Define another struct that implements a trait
struct Student {
    name: String,
    age: u32,
    student_id: String,
}

trait Greet {
    fn greet(&self);
}

impl Greet for Student {
    fn greet(&self) {
        println!("Hello, my name is {} and I am {} years old. My student ID is {}.", self.name, self.age, self.student_id);
    }
}

fn main() {
    // Create an instance of the Person struct
    let person = Person {
        name: String::from("Jane Smith"),
        age: 25,
    };

    // Call methods on the Person instance
    person.greet();

    // Use the default values defined in the Person struct
    let default_person = Person::default();
    default_person.greet();

    // Create an instance of the Student struct
    let student = Student {
        name: String::from("Alice Johnson"),
        age: 20,
        student_id: String::from("S12345"),
    };

    // Call the greet method defined in the Greet trait
    student.greet();
}